string-manipulation-3/src/is-palindromic.js

```diff
@@ -0,0 +1,21 @@
+/* exported isPalindromic */
+// create an empty counter variable
+// remove all spaces from string
+// for each value of string from index 0 to index (length of string divided by 2 rounded down)
+// if string at current index is strictly equal to string at index of (length of string minus i minus 1), increment counter
+// if counter is strictly equal to length of string divided by 2 rounded down, return true
+// else return false
+function isPalindromic(string) {
+  var counter = 0;
+  string = string.split(' ').join('');
```

Comment:
Same deal here, this is two loops to remove spaces, you can do it with one loop if you code it yourself.